.gitmodules

```diff
@@ -13,9 +13,6 @@
 [submodule "external/wypopovercontroller"]
 	path = external/wypopovercontroller
 	url = https://github.com/sammcewan/WYPopoverController
-[submodule "external/react-native"]
```

Comment:
Submodule is gone. We don't need the fork anymore. Future upgrades should be a lot easier (especially since we now have tests).